includes/compats/class-wc-amazon-payments-advanced-multi-currency-abstract.php

```diff
@@ -203,5 +213,24 @@ public function is_order_reference_checkout_suspended() {
 		return false;
 	}
 
+	/**
+	 * Filters the validity of the current checkout_session
+	 *
+	 * @param  bool|WP_Error $valid Is the checkout_session_still_valid?.
+	 * @param  object        $checkout_session Checkout Session Object.
+	 * @return bool|WP_Error
+	 */
+	public function is_checkout_session_still_valid( $valid, $checkout_session ) {
+		if ( is_wp_error( $valid ) ) {
```

Comment:
shouldn't be `if ( ! is_wp_error( $valid ) ) `

Reply:
No, because if it's already a WP_Error, the session is already invalid, so we return it as is.

Reply:
gotcha